tutorials/self-signed-docker-registry-on-gke.md

```diff
@@ -0,0 +1,114 @@
+---
+title: Using a self-signed Docker Registry on GKE
+description: Using a self-signed Docker Registry on GKE.
+author: samos123
+tags: Kubernetes Engine, GKE, Docker Registry
+date_published: 2020-01-28
+---
+
+This tutorial shows how to use a Docker registry that was signed by a 
+non-trusted CA with Google Kubernetes Engine (GKE). Let's assume your company
+has a private CA that's not trusted by default.
+All internal services such as on-prem docker registry have their certs signed
+by this private CA. This would cause GKE to prevent pulling from the on-prem
+registry since it's not trusted. In this guide we assume the private CA cert is
+stored in a file called `myCA.pem`.
+
+The solution is to import the CA into the system bundle and restart docker.
```

Comment:
I would explain system bundle in a sentence or two.
Restart docker where? I would explain clearly that all GKE nodes need to have this done